.github/workflows/pr.yaml

```
@@ -27,31 +28,30 @@ jobs:
       - name: Deploy charts
         timeout-minutes: 5
         run: |
-          kubectl cluster-info
+          kubectl cluster-info -v8
           make deploy-cert-manager
-          make deploy-konk
-      - name: Test charts
-        timeout-minutes: 5
-        run: |
-          make test-konk
       - name: Test konk-operator
         timeout-minutes: 5
         run: |
-          make kind-load-konk KIND_NAME="chart-testing"
-          make deploy-konk-operator
+          make -e kind-load-konk-operator
+          make -e deploy-konk-operator
           kubectl create -f examples/konk.yaml
           until kubectl wait --timeout=3m --for=condition=ready pod -l app.kubernetes.io/component=apiserver,app.kubernetes.io/instance=example-konk
```

It's named `example` now.
```suggestion
          until kubectl wait --timeout=3m --for=condition=ready pod -l app.kubernetes.io/component=apiserver,app.kubernetes.io/instance=example
```

---

Take one down, pass it around, test is still failing.